test/evergreen.yml

```diff
@@ -780,7 +780,6 @@ variables:
   - &configure_flags_tiered_storage_s3
     ENABLE_S3: -DENABLE_S3=1
     IMPORT_S3_SDK: -DIMPORT_S3_SDK=external
-    CMAKE_PREFIX_PATH: # FIXME-WT-9590 - the S3 extension tests core with TCMALLOC.
```

Comment:
Do we not want to set the tcmalloc flag here to enable it for s3 tests? Or is it already a default flag elsewhere?

Reply:
It will be inherited from the variant (this was explicitly disabling it, now it will be up to the variant).

Reply:
That makes sense. Thanks!